### PR TITLE
Switch to actual working @Mojo descriptor from comment-based descriptor

### LINKED ISSUE #30: write-active-profile-properties configuration param outputFile not allowed
maven throws error while processing 

```
<execution>
  <id>write-properties</id>
  <phase>generate-resources</phase>
  <goals>
    <goal>write-active-profile-properties</goal>
  </goals>
  <configuration>
    <outputFile>
     src/main/resources/maven.properties
    </outputFile>
  </configuration>
</execution>
```

With error message 
Execution write-properties of goal org.codehaus.mojo:properties-maven-plugin:1.0.0:write-active-profile-properties failed. NullPointerException

Intellij IDEA shows that Element "outputFile" is not allowed here.

write-project-properties goal works as expected

### PR DESCRIPTION
**Pull request by mykelalvis**
Needs tests.  Fixes #30

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -21,19 +21,19 @@
...
-  <parent>
-    <artifactId>mojo-parent</artifactId>
-    <groupId>org.codehaus.mojo</groupId>
-    <version>38</version>
-  </parent>
+	<parent>
+		<groupId>io.cotiviti</groupId>
+		<artifactId>cotiviti-parent</artifactId>
+		<version>0.3.5-SNAPSHOT</version>
+	</parent>
```
Comment (khmarbaise) [inline pom.xml]: What is the intention of this change?

File: pom.xml [hunk 2]
```diff
@@ -66,12 +66,34 @@
...
 	<scm>
-		<connection>scm:git:https://github.com/cotiviti/cotiviti-properties-maven-plugin.git</connection>
-		<developerConnection>scm:git:ssh://git@github.com/cotiviti/cotiviti-properties-maven-plugin.git</developerConnection>
-		<url>https://github.com/cotiviti/cotiviti-properties-maven-plugin</url>
+		<connection>scm:git:${git.url}</connection>
+		<url>scm:git:${git.url}</url>
+		<developerConnection>scm:git:${git.url}</developerConnection>
 		<tag>HEAD</tag>
 	</scm>
+	<distributionManagement>
```
Comment (khmarbaise) [inline pom.xml]: All things like distributionManagement is already defined in mojo parent..

Comment (mykelalvis) [inline pom.xml]: Yup.  This was definitely my mistak

File: pom.xml (hunk 1 above)
Comment (mykelalvis) [inline pom.xml]: This was inadvertently changed in our fork's `master`.  I will figure it out and re-try.

**Pull request closed without merging**